extensions/3DTILES_implicit_tiling/0.0.0/example/subtree.json

```diff
@@ -0,0 +1,94 @@
+{
+	"buffers": [
+    {
+      "uri": "availability.bin",
+      "byteLength": 100
+    },
+    {
+      "uri": "layers_availability.bin",
+      "byteLength": 100
+    },
+    {
+      "uri": "metadata.bin",
+      "byteLength": 100
+    }
+  ],
+  "bufferViews": [
+    {
+      "buffer": 0,
+      "byteOffset": 0,
+      "byteLength": 100
+    },
+    {
+      "buffer": 0,
+      "byteOffset": "(branchingFactor ^ (subtreeDepth + 1) - 1) / (branchingFactor - 1) / bitsPerByte",
```

Comment:
Just a note for the future: these formulas would be helpful for the spec, but as far as the example goes, I'd rather see a concrete example (doesn't have to be this PR)